spyder/utils/qstringhelpers.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+# (see spyder/__init__.py for details)
+
+"""QString compatibility."""
+
+from spyder.py3compat import PY2
+
+
+def qstring_length(text):
```

Comment:
Is there a reason why you couldn't move this to `qthelpers.py`, instead of creating a new module?

Reply:
`qthelper` imports `spyder.utils.icon_manager` which imports `spyder.config.gui` which imports `spyder.utils.syntaxhighlighters` which imports `qstring_length`. Therefore `spyder.utils.syntaxhighlighters` can't import `qthelper` without creating an import loop.

Reply:
Alternatively, I can put `qstring_length` before the imports in `qthelper`

Reply:
Alternatively, I can put this file here: https://github.com/jupyter/qtconsole/pull/366

Reply:
Ok, as I thought, this was a problem with circular imports. Thanks for the clarification :+1: 

And it's fine to leave it here.